### DISCUSSION TITLE
Americans with Disabilities Act (ADA) guidelines

### DISCUSSION THREAD

**Discussion by CharlyAutomatico** [Purfview/whisper-standalone-win#246] [Q&A] [answered]
Hello, is it possible to limit the number of characters or the duration of each time scale? I need to comply with the Americans with Disabilities Act (ADA) guidelines.

Thanks

**Comment by Purfview** [comment, marked as answer]
> Hello, is it possible to limit the number of characters

Use --max_line_width with --sentence, or just --standard.
